src/main/java/com/synopsys/integration/alert/channel/rest/ChannelRestConnectionFactory.java

```diff
@@ -51,72 +50,33 @@ public ChannelRestConnectionFactory(final AlertProperties alertProperties) {
         this.alertProperties = alertProperties;
     }
 
-    public RestConnection createUnauthenticatedRestConnection(final String stringUrl) {
-        final URL url = getUrlFromString(stringUrl);
-        return createUnauthenticatedRestConnection(url);
+    public RestConnection createRestConnection() {
+        return createRestConnection(new Slf4jIntLogger(logger), 5 * 60 * 1000);
     }
 
-    public RestConnection createUnauthenticatedRestConnection(final URL url) {
-        return createUnauthenticatedRestConnection(url, new Slf4jIntLogger(logger), 5 * 60 * 1000);
-    }
-
-    public RestConnection createUnauthenticatedRestConnection(final URL url, final IntLogger intLogger, final int timeout) {
-        if (url == null) {
-            logger.error("URL WAS NULL");
-            return null;
-        }
-        final UnauthenticatedRestConnectionBuilder restConnectionBuilder = createUnauthenticatedRestConnectionBuilder(intLogger, url.toString(), timeout);
-
-        final RestConnection connection = restConnectionBuilder.build();
-        try {
-            // the build operation will catch the issues based on the configuration settings and throw an exception
-            // the IntegrationException caught here is unlikely to occur with an UnauthenticatedRestConnection.
-            connection.connect();
-            return connection;
-        } catch (final IntegrationException e) {
-            logger.error("Could not connect to " + url.toString(), e);
-            return null;
-        }
-    }
-
-    private URL getUrlFromString(final String apiUrl) {
-        URL url = null;
-        try {
-            url = new URL(apiUrl);
-        } catch (final MalformedURLException e) {
-            logger.error("Problem generating the URL: " + apiUrl, e);
-        }
-        return url;
+    public RestConnection createRestConnection(final IntLogger intLogger, final int timeout) {
+        final Optional<Boolean> alertTrustCertificate = alertProperties.getAlertTrustCertificate();
```

Comment:
Instead of allowing this to be set to true, should we just default it to false since our stance is they should import their certs?

Reply:
I would leave this here.